packages/cozy-harvest-lib/src/datacards/ViewerModal.jsx

```diff
@@ -8,10 +8,9 @@ import { useDataCardFiles } from './useDataCardFiles'
 import { MountPointContext } from '../components/MountPointContext'
 
 export const ViewerModal = () => {
+  const { accountId, folderToSaveId, fileIndex } = useParams()
   const { pushHistory, replaceHistory } = useContext(MountPointContext)
   const { data, fetchStatus } = useDataCardFiles(accountId, folderToSaveId)
```

Comment:
how our linter didn't tell us about this error?

Reply:
I'm not sure the linter could have helped here (can't think of a rule). But TypeScript would have detected that the parameters could be undefined

Reply:
accountId was not defined yet. So linter should have been able to tell us that we were reading a non created variable yet.

Reply:
Yes, that's correct, maybe the linter was confused by react-specific rules and thought it was hoisted or something